django/apps/registry.py

```diff
@@ -260,6 +260,27 @@ def get_registered_model(self, app_label, model_name):
                 "Model '%s.%s' not registered." % (app_label, model_name))
         return model
 
+    # This method is performance-critical when it comes to migrations.
```

Comment:
I would put this in the docstring "This method is decorated with lru_cache because it's performance critical..."